includes/Core/Tags/Guards/Tag_Environment_Type_Guard.php

```diff
@@ -31,6 +31,16 @@ public function can_activate() {
 		if ( ! function_exists( 'wp_get_environment_type' ) ) {
 			return true;
 		}
-		return 'production' === wp_get_environment_type();
+
+		$allowed_environments = apply_filters(
+			'googlesitekit_allowed_tag_environment_types',
+			array( 'production' )
+		);
+
+		if ( ! is_array( $allowed_environments ) ) {
+			$allowed_environments[] = 'production';
+		}
```

Comment:
If `$allowed_environments` is not an array then we can't treat it as an array. We need to assign an array with the `production` value instead:
```suggestion
		if ( ! is_array( $allowed_environments ) ) {
			$allowed_environments = array( 'production' );
		}
```